packages/client/mutations/handlers/handleUpdateSpotlightResults.ts

```diff
@@ -29,13 +29,13 @@ const handleUpdateSpotlightResults = (
     oldReflections?.length === 1 && oldReflections[0].getValue('id') === reflectionId
   const reflectionGroupId = reflectionGroup.getValue('id')
   const groupsIds = similarReflectionGroups.map((group) => group.getValue('id'))
-  const isInSpotlight = groupsIds.includes(reflectionGroupId)
-  const wasInSpotlight = groupsIds.includes(oldReflectionGroupId)
+  const isInSpotlight = [...groupsIds, spotlightGroupId].includes(reflectionGroupId)
```

Comment:
instead of creating the same array twice just concat spotlightGroupId into groupIds above